src/test/scala/io/qbeast/spark/keeper/ProtocolMockTest.scala

```diff
@@ -4,7 +4,7 @@ import io.qbeast.core.keeper.{Keeper, LocalKeeper}
 
 class ProtocolMockTest extends ProtocolMockTestSpec {
   "the qbeast-spark client" should
-    "throw an execution when an inconstant state is found" in withContext(LocalKeeper) {
+    "throw an execution when an inconstant state is found" ignore withContext(LocalKeeper) {
```

Comment:
I think we can delete these tests now, as we don't have conflicts anymore (just a replication will need to be re-executed). What do you think @osopardo1 ?

Reply:
This (and another one similar) test checks if two concurrent operations lead to conflict, however with the new code the conflict does not occur because we do not write the replicated cubes into the metadata any longer.

Reply:
If the conflict those not occur, let's delete them.

Reply:
done